### PR TITLE
Add cpu and memory limit to Pod builder

### PR DESCRIPTION
**Pull request by gkopels**

### CODE REVIEW

File: pkg/clients/clients.go [hunk 1]
```diff
@@ -212,6 +217,14 @@ func SetScheme(crScheme *runtime.Scheme) error {
 		return err
 	}
 
+	if err := nmstatev1.AddToScheme(crScheme); err != nil {
```
Comment (kononovn) [inline pkg/clients/clients.go:220]: According to commit message it's not related to your PR

File: pkg/pod/pod.go [hunk 2]
```diff
@@ -487,6 +489,32 @@ func (builder *Builder) WithTolerationToMaster() *Builder {
...
+	builder.isMutationAllowed("resource spec")
+
+	if builder.errorMsg != "" {
+		return builder
+	}
+
+	for index := range builder.Definition.Spec.Containers {
+		builder.Definition.Spec.Containers[index].Resources = v1.ResourceRequirements{
+			Limits: v1.ResourceList{
+				v1.ResourceCPU:    resource.MustParse(strconv.Itoa(2)),
```
Comment (kononovn) [inline pkg/pod/pod.go:509]: It shouldn't be hard-coded

File: pkg/clients/clients.go (hunk 1 above)
Comment (gkopels) [reply pkg/clients/clients.go:220]: I didnt make any changes to this file. I believe I incorrectly created a commit including the previous PR. Need to remove.

**Comment by kononovn** [comment]
closing PR because relevant tests are going to be added upstream.

**Pull request closed without merging**